.github/workflows/proto.yml

```
@@ -28,6 +28,7 @@ jobs:
         with:
           go-version: 1.15
       - uses: ./.github/actions/generate-swagger
+      - run: git diff-index --quiet HEAD
```

Can you add this? If it does not produce any clue, I will ask you later to remove this line as you did.
```suggestion
      - run: git diff-index --quiet HEAD
      - name: report
        run: git diff
        if: ${{failure()}}
```